### PR TITLE
Add function for adding descriptions

### PR DESCRIPTION
**Pull request by lokal-profil**
Task: https://phabricator.wikimedia.org/T159320

### CODE REVIEW

File: wikidataStuff/WikidataStuff.py [hunk 1]
```diff
@@ -213,6 +213,32 @@ def searchGenerator(self, text, language):
...
+        Add a description to the item in the given language.
+
+        @param lang: the language code
+        @param description: the value to be added, set to an empty string to,
+            remove an existing description.
+        @param item: the item to which the description should be added
+        @param overwrite: whether any pre-existing description should be
+            overwritten.
+        @param summary: summary to append to auto-generated edit summary
+        @param caseSensitive: if the comparison is case sensitive
```
Comment (Vesihiisi) [inline wikidataStuff/WikidataStuff.py]: caseSensitive seems to be missing in the function definition.

Comment (lokal-profil) [inline wikidataStuff/WikidataStuff.py]: Nice spot: Accidentally copy-pasted. Now removed

**Comment by Vesihiisi** [comment]
Just thought about a thing: is there a protection (either in this library or in Pywikibot... or in Wikidata itself I guess) against adding zero-length strings as descriptions/labels? What WOULD happen if you passed such a string to this script?

**Comment by lokal-profil** [comment]
By passing a zero length string you remove the value from wikidata (it's documented in pywikibot but not super easy to find)

**Comment by Vesihiisi** [comment]
Right, that makes sense.

**Comment by Vesihiisi** [comment]
But I guess this looks good? If you merge this, I can update my Uploader to use it.

**Pull request merged**